language_service/src/completion/tests.rs

```diff
@@ -1,9 +1,36 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+use expect_test::{expect, Expect};
+
 use super::{get_completions, CompletionItemKind};
 use crate::test_utils::{compile_with_fake_stdlib, get_source_and_marker_offsets};
 
+fn assert_completions_contain(
+    source_with_cursor: &str,
+    completions_to_check: &[&str],
+    expect: &Expect,
```

Comment:
I love the refactor to make `expect` work!

I'm also not opposed to renaming the function to `check()` or something since it's what we've been going with for other unit tests (like hover)